docs/using-full-scan-and-full-partition-scan-in-longevity.md

```diff
@@ -0,0 +1,104 @@
+### **Guide: Using Full-Scan and Full-Partition-Scan in Scylla Cluster Test (SCT)**
+
+**Overview**
+In Scylla Cluster Test (SCT) Longevity, `full-scan` and `full-partition-scan` are background threads used for testing long-duration select-queries. These operations are configured in test YAML files and help ensure that the database’s tables and partitions are consistent and can be read without unexpected Scylla errors.
+
+This can be tested during or after various disruptive nemesis.
+
+* **Full-scan**: Its primary goal is to verify that Scylla can handle a comprehensive scan of an entire table without significant performance issues or failures.
+  * Variants:
+    * A simple “select \*” from a table.
+    * An aggregation mode of “select count(\*)”
+* **Full-partition-scan**: Particularly relevant for large-partition test configurations, this scan focuses on counting the number of rows per partition.
+  * variants:
+    * Read all partition data
+    * Query and compare a partition data in both Ascending and Descending order  (by the parameter of "validate\_data")
+
+**Configuration in YAML**
+The full-scan operation can be customized by specifying parameters like mode, keyspace, table, and intervals. The two main modes are:
+
+* **Full Table Scan (`table`)**: This scans all the rows in the specified table.
+* **Full Partition Scan (`partition`)**: This scans partitions, particularly useful for testing large partitions or checking partition key ranges.
+
+You can configure full-scan operations in your test YAML by defining a `run_fullscan` parameter with a set of attributes. Below is an example of a typical YAML configuration:
+
+run\_fullscan: \['{"mode": "table\_and\_aggregate", "ks\_cf": "keyspace1.standard1", "interval": 10}'\]
+
+This example performs a full table scan and aggregates the results from the keyspace `keyspace1` and table `standard1` every 10 seconds.
+
+#### **Common Parameters:**
+
+1. **`mode`**: Defines the type of scan. Can be `table`, `aggregate,` `partition`, or `table_and_aggregate`.
```

Comment:
Can be `table`, `aggregate,` `partition`
 random - rand of 3 
 `table_and_aggregate - rand of table`, `aggregate

Reply:
fixed.